packages/types/src/DidDetails.ts

```diff
@@ -10,10 +10,19 @@ import type { BN } from '@polkadot/util'
 import type { DidPublicKey } from './DidDocumentExporter'
 import type { IIdentity } from './Identity'
 
+type Digit = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'
+
+/**
+ * A KILT DID identifier, e.g., 4nvZhWv71x8reD9gq7BUGYQQVvTiThnLpTTanyru9XckaeWa.
+ */
+export type DidIdentifier = IIdentity['address']
+
 /**
- * A DID identifier, e.g., 4nvZhWv71x8reD9gq7BUGYQQVvTiThnLpTTanyru9XckaeWa.
+ * A string containing a KILT DID Uri.
  */
-export type IDidIdentifier = IIdentity['address']
+export type DidUri =
+  | `did:kilt:${DidIdentifier}`
+  | `did:kilt:light:${Digit}${Digit}${DidIdentifier}${string}`
```

Comment:
I would just add a comment both here and where the regex is, that the two has to be in sync with each other.

Reply:
Plus, there is an additional `:` between a light DID identifier and the string representing the encoded details.

Reply:
done!